src/components/Camera/index.js

```diff
@@ -181,7 +184,7 @@ export default class Camera extends React.Component<CameraType, CameraStateType>
     }
 
     if (this.props.autoCapture) return <AutoCapture {...props} />
-    return process.env.LIVENESS_ENABLED && this.props.variant === 'video' ?
+    return shouldUseLiveness(this.props.variant) ?
```

Comment:
feels a bit weird that this determination is made in two places

Reply:
why not have a `LivenessCapture` and `LivenessConfirm`, since part of this determination is done at router level? (I understand this might be seen as out of scope)

Reply:
It does feel weird, but
- on the router level we need to decide whether we include or don't include the liveness intro. 
- on the camera level we need to decide whether we should take a selfie or record a video. 

Even if we had a LivenessCapture, it would still be a wrapper of Capture, that holds a good amount of functionality that we need:

- dispatching actions to the store
- handling file upload
- displaying privacy policy

And this `Capture` child will render a `Camera` child, that needs to decide again whether it's a document autocapture, a video recorder or a selfie camera.

Reply:
I think I understand now your idea @rfreitas , will try this in a different way